src/test/kotlin/no/nav/tilleggsstonader/sak/infrastruktur/mocks/JournalpostClientConfig.kt

```diff
@@ -35,180 +36,186 @@ class JournalpostClientConfig {
                         ),
-                        DokumentInfo(
-                            dokumentInfoId = "2",
-                            tittel = "Dummy dokument 2",
-                            dokumentvarianter = listOf(
-                                Dokumentvariant(
-                                    variantformat = Dokumentvariantformat.ARKIV,
-                                    saksbehandlerHarTilgang = true,
-                                ),
+                        dokumentvarianter = listOf(
+                            Dokumentvariant(
+                                variantformat = Dokumentvariantformat.ARKIV,
+                                saksbehandlerHarTilgang = true,
                             ),
                         ),
                     ),
-                ),
-                Journalpost(
-                    "1",
-                    Journalposttype.I,
-                    journalstatus = Journalstatus.MOTTATT,
-                    tema = Tema.TSO.toString(),
-                    behandlingstema = "ab0300",
-                    tittel = "Søknad om barnetilsyn",
-                    bruker = Bruker("12345678910", BrukerIdType.FNR),
-                    avsenderMottaker = avsenderMottaker(),
-                    journalforendeEnhet = "tilleggsstonader-sak",
-                    relevanteDatoer = listOf(
-                        RelevantDato(osloNow().minusDays(7), "DATO_REGISTRERT"),
-                        RelevantDato(osloNow(), "DATO_JOURNALFOERT"),
-                    ),
-                    dokumenter = listOf(
-                        DokumentInfo(
-                            dokumentInfoId = "3",
-                            tittel = "Dummy dokument 3",
-                            logiskeVedlegg = listOf(
-                                LogiskVedlegg("1", "Dokumentasjon på sykdom"),
-                                LogiskVedlegg("2", "Inntektsendring"),
-                                LogiskVedlegg("3", "Samværsmelding"),
-                            ),
-                            dokumentvarianter = listOf(
-                                Dokumentvariant(
-                                    variantformat = Dokumentvariantformat.ARKIV,
-                                    saksbehandlerHarTilgang = true,
-                                ),
+                    DokumentInfo(
+                        dokumentInfoId = "2",
+                        tittel = "Dummy dokument 2",
+                        dokumentvarianter = listOf(
+                            Dokumentvariant(
+                                variantformat = Dokumentvariantformat.ARKIV,
+                                saksbehandlerHarTilgang = true,
                             ),
                         ),
                     ),
                 ),
             )
-        }
 
-        return journalpostClient
-    }
+        fun resetOgInitialisereMocks(journalpostClient: JournalpostClient) {
```

Comment:
Kanskje vi bare kan kalle den for `resetMock´?  Likt som i 
```suggestion
        fun resetMock(client: InfotrygdReplikaClient)
```